gcc/jit/jit-target.h

```diff
@@ -55,6 +56,7 @@ struct target_info {
     std::unordered_map<const char *, std::unordered_set<const char *, CStringHash, CStringEqual>, CStringHash, CStringEqual> m_info;
     std::string m_arch;
     bool m_supports_128bit_int = false;
+    std::unordered_map<enum gcc_jit_types, enum gcc_jit_types> m_supported_target_dependent_types;
```

Comment:
It seems like this could be a `std::unordered_set`.

Reply:
Fixed